src/collection.js

```diff
@@ -28,10 +28,10 @@ export default class Collection {
 	/**
 	 * Creates a new Collection instance.
 	 *
-	 * @param {Object} options The options object.
+	 * @param {Object} [options={}] The options object.
 	 * @param {String} [options.idProperty='id'] The name of the property which is considered to identify an item.
 	 */
-	constructor( options ) {
+	constructor( { idProperty = 'id' } = {} ) {
```

Comment:
The con of this approach is that you're losing the JSDoc comment to param connection. I'm not sure how JSDoc behaves in such moments.

Reply:
I'd say that this should be reverted as this isn't part of this ticket anyway ;)

Reply:
Hi, @Reinmar! How are your holidays?

Reply:
I actually start them in a couple hours. You can see this by the weather ;)

![image](https://user-images.githubusercontent.com/156149/31881320-787913b2-b7e3-11e7-9b0f-79a5ced39c2d.png)

Reply:
I changed it because of
![image](https://user-images.githubusercontent.com/1057056/31881491-f16f56b4-b7e3-11e7-979c-e8c27be695aa.png)
but looks like I changed too much :)

Reply:
Fixed, not reverted but fixed :)